recipes/librdkafka/all/conanfile.py

```diff
@@ -117,6 +114,16 @@ def package_info(self):
         self.cpp_info.components["rdkafka"].names["cmake_find_package_multi"] = "rdkafka"
         self.cpp_info.components["rdkafka"].names["pkg_config"] = "rdkafka"
         self.cpp_info.components["rdkafka"].libs = ["rdkafka"]
+        if self.options.zlib:
+            self.cpp_info.components["rdkafka"].requires.append("zlib::zlib")
+        if self.options.zstd:
+            self.cpp_info.components["rdkafka"].requires.append("zstd::zstd")
+        if self.options.ssl:
+            self.cpp_info.components["rdkafka"].requires.append("openssl::openssl")
+        if self.options.sasl and self.settings.os != "Windows":
+            self.cpp_info.components["rdkafka"].requires.append("cyrus-sasl::cyrus-sasl")
```

Comment:
I think it would be easier to follow it this option was deleted, instead of always using if not windows.

Reply:
Actually, this option is not deleted in recipe.

Reply:
I apologize, my typo made it unclear. I am asking if it should be deleted on windows?

Reply:
It should not be deleted. The option exists on windows and uses Windows libraries to provide this features.

Reply:
Which windows libraries? Does windows system libs deserve some refinements in `package_info`?

```python
    def package_info(self):
        ...
        if self.settings.compiler == "Visual Studio":
            self.cpp_info.components["rdkafka"].system_libs.extend(["crypt32", "ws2_32"])
        ...
```

Reply:
It uses secur32 linked with #pragma comment https://github.com/edenhill/librdkafka/blob/master/src/rdkafka_sasl_win32.c#L46.

Reply:
Should be added in system libs if Windows, for MinGW at least (pretty sure that this pragma is ignored in MinGW).